src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.ts

```diff
@@ -340,7 +371,19 @@ export class RgwBucketFormComponent extends CdForm implements OnInit, AfterViewC
       // make the owner empty if the field is disabled.
       // this ensures the bucket doesn't gets updated with owner when
       // the bucket is owned by the account.
-      const owner = this.bucketForm.get('owner').disabled === true ? '' : values['owner'];
+      let owner;
+      if (this.bucketForm.get('accountUser').disabled === true) {
+        // If the bucket is owned by the account, then set the owner as account user.
+        owner = '';
```

Comment:
i didn't get this btw, how is it deduced that the bucket is owned by the account when the accountUser field is disabled? or is the comment needs to be somewhere else?